RecoMuon/MuonSeedGenerator/src/MuonSeedOrcaPatternRecognition.cc

```diff
@@ -296,14 +404,25 @@ void MuonSeedOrcaPatternRecognition::endcapPatterns(
   rememberCrackSegments(me3,  crackSegments);
   rememberCrackSegments(me4,  crackSegments);
 
+  rememberCrackSegments(ge21, crackSegments);
+  rememberCrackSegments(ge11, crackSegments);
+  rememberCrackSegments(me0, crackSegments);
```

Comment:
shouldn't the crackEtas be updated for ME0 above eta of 2.4.
Also, do we really get useful gain from single-gem segments in these cracks?